docs/index.md

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
Are the indexes not automatically generated?

Reply:
@brian-brazil not sure about it. I didn't get the idea that the index.md on the promo repo was generated. Is there any reference about this?

Reply:
If you look at them they're just the section title, with nothing else.

Reply:
@grobie how did you generate the index.md on the prometheus doc?

Reply:
I wrote the `index.md` by hand actually. My intent was to give users who read the docs locally a starting point. It was not meant to be just a list of the pages, but to include a small introduction as well. We can change that, but I guess a little explanation that the main documentation lives on a page online would be nice.

Reply:
@brian-brazil @grobie should I add a brief content to it, or just remove it for now?

Reply:
I'd keep it as starting point. But I'm fine with either.